spec/vehicle_factory_spec.rb

```diff
@@ -0,0 +1,66 @@
+require './spec/spec_helper'
+
+RSpec.describe VehicleFactory do
+  before(:each) do
+    @factory = VehicleFactory.new
+    @dds = DmvDataService.new
+  end
+
+  it 'exists' do
+    expect(@factory).to be_an_instance_of(VehicleFactory)
+  end
+
+  it 'works with fake vehicles' do
+    cars = [
```

Comment:
Love that you used a smaller dataset here - this data can live in its own file next time.